### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by dissco6**

### CODE REVIEW

File: front-end-side/src/Pages/Data.css [hunk 1]
```diff
@@ -173,25 +173,50 @@ table.data_table {
...
+    }
+
+    table.data_table td:nth-of-type(3) {
+        max-width: 20vw!important;
+    }
+}
+
+@media only screen and (max-width: 600px) {
+    table.data_table td.edit, table.data_table td.move, table.data_table td.delete, table.data_table th.last {
+        display: none;
```
Comment (Vaivaal) [inline front-end-side/src/Pages/Data.css:219]: Why did we stop displaying those at 600px?

Comment (dissco6) [reply front-end-side/src/Pages/Data.css:219]: It is not so neccessary to see them all the time, especialy when we don't have much space (in mobile view). In the future we will make those buttons apear after clicking on a particular record.

File: front-end-side/src/Components/Modal.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const Modal = props => {
     }, [])
 
     function removeClass() {
-        document.getElementsByTagName("BODY")[0].setAttribute("class", ""); //FIX ME
+        document.getElementsByTagName("BODY")[0].setAttribute("class", "modal-open"); //FIX ME
```
Comment (Vaivaal) [inline front-end-side/src/Components/Modal.js:21]: The comment is not necessary

File: front-end-side/src/Pages/Data.js [hunk 3]
```diff
@@ -185,8 +189,10 @@ const EnterData = () => {
             });
 
             setRerender(true);
+
+            console.log(previousYearExpenses);
```
Comment (Vaivaal) [inline front-end-side/src/Pages/Data.js:193]: Not necessary

Comment (dissco6) [reply front-end-side/src/Pages/Data.js:193]: uhhh

File: front-end-side/src/Pages/Data.js [hunk 4]
```diff
@@ -142,6 +142,10 @@ const EnterData = () => {
 
     useEffect(() => {
         if (categoryStatus) {
+
+            setPreviousMonthExpenses(categoryData.data.previousCarSum + categoryData.data.previousClothesSum + categoryData.data.previousEntertaintmentSum + categoryData.data.previousFoodSum + categoryData.data.previousOtherSum + categoryData.data.previousHouseSum);
+            setPreviousYearExpenses(categoryData.data.previousYearCarSum + categoryData.data.previousYearClothesSum + categoryData.data.previousYearEntertaintmentSum + categoryData.data.previousYearFoodSum + categoryData.data.previousYearOtherSum + categoryData.data.previousYearHouseSum);
```
Comment (Vaivaal) [inline front-end-side/src/Pages/Data.js:147]: We have ```setPreviousMonthExpenses``` and ```setPreviousYearExpenses``` in ```sendDate``` function, are they needed there?

Comment (dissco6) [reply front-end-side/src/Pages/Data.js:147]: Oh, I missed it, thank u

**Pull request merged**